DevelopmentProposals/reporterColumnLabels/reporterColumnLabels.md

```diff
@@ -54,14 +54,15 @@ and forcing the user to specify an annotation for each Output may not be practic
 - The TableReporter should use a default column name that is very likely to be meaningful.
 
 ### Architecture
-The selected design involves four main changes:
+The selected design involves six main changes:
 
 1. Rename "annotation" to "alias" (e.g., `AbstractInput::getAnnotation()` becomes `AbstractInput::getAlias()`).
 2. If the user provides the optional second argument when connecting an Output to an Input, store the string as the Input's alias; otherwise, the alias remains null.
-3. Create `getShortLabel()` and `getLongLabel()` methods on Input:
-  - `getShortLabel()` will be called by ConsoleReporter and will return `isNull(alias) ? output_name : alias`.
-  - `getLongLabel()` will be called by TableReporter and will return `isNull(alias) ? full_path_name : alias`.
-4. Create `setAlias()` methods analogous to the current `getAnnotation()` methods so the user can change the alias after (but not before) connecting.
+3. Create `getReportedLabel()` on Input, which will always return the alias if one is provided:
+  - `getReportedLabel()` will be called by a Reporter and will return `isNull(alias) ? full_path_name : alias`.
```

Comment:
> you demanded I update the DevProp, which I did. A 15-20min direct conversation could have established agreement to revise/or not

The whole Dev Team was involved in selecting the design, so we cannot decide to change it without everyone's involvement. I think one of the objectives of the DevProps is to make the design process more transparent and inclusive.

> we should all feel free to discuss any concerns we have at anytime in the dev process

I disagree. The DevProp was written ~3 weeks before the meeting and we spent an hour discussing designs. This is not a critical piece of the software, and it seems like re-considering the design is not an ideal allocation of resources leading up to Friday's alpha release. Re-designing aliases takes time away from other issues that have not been addressed at all.